config/locales/email_subscribers.de.yml

```
@@ -0,0 +1,41 @@
+de:
+  email_subscribers:
+    form:
+      subscribe_description: "Abonniere den Newsletter, um wöchentlich eine Zusammenfassung der Beiträge in Deinem Posteingang zu erhalten"
```

```suggestion
      subscribe_description: "Abonniere den Newsletter, um wöchentlich eine Zusammenfassung der Beiträge in deinem Posteingang zu erhalten"
```

keep spelling consistent